compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/layout/Layout.kt

```diff
@@ -134,6 +134,16 @@ inline fun Layout(
     )
 }
 
+/*
+ * TODO: This is a version of Layout not using SAM-converted [MeasurePolicy] interface.
```

Comment:
There is a chance, that this will break binary compatibility (because it is a new API). We can't release it in 1.2.0, so we should be figure out what is going on here before the release.

Reply:
For alpha this fix is okay.

Reply:
This code was added first time on 13 April 2022:
https://github.com/JetBrains/androidx/commit/a496f4c68459907d9f1a5d88fbca0bd37ce618cd

@igordmn  Did we release this code in version 1.1.0 ?